tests/test_add_losses.py

```diff
@@ -11,7 +11,10 @@ def test_add_losses():
 
     spectrum = add_losses(spectrum_in)
 
-    assert numpy.allclose(spectrum.losses.mz, numpy.array([145, 245, 295, 345], "float"))
+    expected_mz = numpy.array([145, 245, 295, 345], "float")
+    assert numpy.allclose(spectrum.losses.mz, expected_mz), "Expected different loss m/z."
+    expected_intensities = numpy.array([1000, 100, 200, 700], "float")
```

Comment:
please move this to just after line 14

Reply:
Sorry, I am not sure what you mean here.

Reply:
changed the order. hope that's what you meant?

Reply:
yes, thanks